zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -1593,7 +1586,7 @@ paths:
         content:
           application/json:
             schema:
-              $ref: "#/components/schemas/CreateProcessInstanceRequest"
+              $ref: "#/components/schemas/CreateProcessInstanceRequestBody"
```

Comment:
🔧 The example below also needs to move to using a string key instead of a number

Reply:
❓ @pepopowitz, do I recall correctly that we have an automation in `camunda-docs` regarding this endpoint? Does that need an adjustment now?

Reply:
Ah right, the example! I'll update it.